src/site/content/en/blog/async-clipboard/index.md

```diff
@@ -129,37 +129,45 @@ try {
 
 ### The copy event
 
-In the case where a user initiates a clipboard copy, non-textual data is
-provided as a Blob for you. The
+In the case where a user initiates a clipboard copy, the
 [`copy` event](https://developer.mozilla.org/docs/Web/API/Document/copy_event)
-includes a `clipboardData` property with the items already in the right format,
-eliminating the need to manually create a Blob. Call `preventDefault()` to
-prevent the default behavior in favor of your own logic, then copy contents to
-the clipboard. What's not covered in this example is how to fall back to earlier
+includes a `clipboardData` property with the items already in the right format.
+
```

Comment:
```suggestion
includes a `clipboardData` property with the items already in the right format.
```

Reply:
This feels like background information for the next paragraph. Since it's so short, just make it part of the next paragraph. 

I think this property might read only in Firefox. If I'm correctly remembering which bit of code I was stepping through, I was trying to manipulate this in the Firefox console and not having any luck.

Reply:
I'll merge the two paragraphs. 

It also reads in Chrome, but not when you `preventDefault()`.